lib/rubycritic/core/analysed_module.rb

```diff
@@ -19,6 +23,22 @@ def path
       @path ||= pathname.to_s
     end
 
+    def file_location
+      file_path[0...-1].join('/')
```

Comment:
It should support Windows (at least I think it used to).

And `File::SEPARATOR` is unnecessary as an `analysed_module` has a `pathname` attribute which is of the type, uh, [Pathname](http://ruby-doc.org/stdlib-1.9.3/libdoc/pathname/rdoc/Pathname.html) 😄 

This method should simple be

```
pathname.dirname
```